src/components/Controls/Select.vue

```diff
@@ -38,7 +37,7 @@
       </select>
       <svg
         class="w-3 h-3"
-        style="background: inherit; z-index: 1; transform: translateX(3px)"
+        style="background: inherit; margin-right: -3px"
```

Comment:
What are these changes for?

Reply:
Same because of the pull. These are some style changes for Select drop-down, updated by Alan.